### DISCUSSION TITLE
Any plan to introduce the `refresh_token` scheme?

### DISCUSSION THREAD

**Discussion by retzero** [fastapi/full-stack-fastapi-template#1441] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
No.
```

### Description

When the `access_token` expires, the user encounters `403` error.
Do you have plan to introduce `refresh_token` scenario also?

### Operating System

Linux

### Operating System Details

Ubuntu 22.04

### Python Version

3.10.12

**Comment by tiangolo** [comment, marked as answer]
A refresh token wouldn't add any extra security as the final public client would be the same one that gets access to it. A refresh token would make more sense in other OAuth2 flows, like a code flow, where a secure backend can hold the refresh token.
